src/lib/Menu/ContentRightSidebarBuilder.php

```diff
@@ -388,19 +406,70 @@ private function addRevealMenuItem(ItemInterface $menu): void
     /**
      * @param \Knp\Menu\ItemInterface $menu
      */
-    private function addHideMenuItem(ItemInterface $menu): void
+    private function addHideMenuItem(ItemInterface $menu, bool $canHide): void
     {
+        $attributes = [
+            'data-actions' => 'hide',
+            'class' => 'ez-btn--hide',
+        ];
+
         $menu->addChild(
             $this->createMenuItem(
                 self::ITEM__HIDE,
                 [
                     'extras' => ['icon' => 'hide'],
-                    'attributes' => [
-                        'data-actions' => 'hide',
-                        'class' => 'ez-btn--hide',
-                    ],
+                    'attributes' => $canHide
+                        ? $attributes
+                        : array_merge($attributes, ['disabled' => 'disabled']),
                 ]
             )
         );
     }
+
+    private function canCreateAnywhere(Content $content): bool
+    {
+        $createPolicies = $this->permissionResolver->hasAccess(
+            'content',
+            'create'
+        );
+
+        if (is_bool($createPolicies)) {
+            return $createPolicies;
+        }
+
+        foreach ($createPolicies as $createPolicy) {
+            $sectionId = $content->contentInfo->sectionId;
+            if (
+                !empty($createPolicy['limitation']) &&
+                $createPolicy['limitation'] instanceof SectionLimitation &&
+                !in_array($sectionId, $createPolicy['limitation']->limitationValues)
+            ) {
+                return false;
+            }
+
+            if (empty($createPolicy['policies'])) {
+                return true;
+            }
+
+            foreach ($createPolicy['policies'] as $policy) {
```

Comment:
`foreach` loop will in practice skip an empty array.

```suggestion
            foreach ($createPolicy['policies'] ?? [] as $policy) {
```